### PR TITLE
[UNKNOWN] generic exit page

### PR DESCRIPTION
**Pull request by DrogoNevets**
- **exit page started**
- **trying to get tests working --no-verify**
- **trying to get tests working**
- **updated tests for branch**
- **change title size**

### CODE REVIEW

File: src/server/common/constants/status-codes.js [hunk 1]
```diff
@@ -8,5 +8,6 @@ export const statusCodes = {
   unauthorized: 401,
   forbidden: 403,
   notFound: 404,
-  imATeapot: 418
+  imATeapot: 418,
+  redirect: 302
```
Comment (hughfdjackson) [inline src/server/common/constants/status-codes.js:12]: I _think_ it's intuitive that this is the temporary redirect (and that we call `301` `permanantRedirect` in the future).  Do you think there's any scope for confusion?

File: src/server/cph-number/controller.js [hunk 2]
```diff
@@ -28,7 +28,10 @@ export const postController = {
   handler(req, res) {
     const valid = /([0-9]{2})\/([0-9]{3})\/([0-9]{4})/g
     let errorMessage = null
-    const { cphNumber } = /** @type {CphNumberPayload} */ (req.payload)
+    let { cphNumber } = /** @type {CphNumberPayload} */ (req.payload)
+
+    // Remove whitespace from cphNumber
+    cphNumber = cphNumber ? cphNumber.replace(/\s+/g, '') : ''
```
Comment (hughfdjackson) [inline src/server/cph-number/controller.js:34]: happy to pull it through as one this time, but would prefer a separate PR for this typically.

File: src/server/exit-page/index.njk [hunk 3]
```diff
@@ -0,0 +1,35 @@
+{% extends 'layouts/page.njk' %}
+
+{% block content %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <div class="govuk-body" data-testid="app-page-body">
+        <div class="gem-c-title govuk-!-margin-top-0 govuk-!-margin-bottom-8">
```
Comment (hughfdjackson) [inline src/server/exit-page/index.njk:8]: In the prototype, we're missing this wrapper & I can't see it in the [GDS design system](https://design-system.service.gov.uk/styles/headings/).  

Worth removing?

**Review by hughfdjackson — CHANGES_REQUESTED** [review]
Just one real change here - the styling on the header.

File: src/server/cph-number/controller.js (hunk 2 above)
Comment (hughfdjackson) [reply src/server/cph-number/controller.js:34]: @eoin-corr-git  - are we looking to add a test for this case in the user journey tests?

File: src/server/common/constants/status-codes.js (hunk 1 above)
Comment (DrogoNevets) [reply src/server/common/constants/status-codes.js:12]: my concern over 301 was that it redirects to different places depending on the input (`payload`) and thus I thought it was more a 302 than a 301

Also looking at [MDN 302](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/302) = Found, not temp (which was news to me, but why i chose it over 300)

File: src/server/cph-number/controller.js (hunk 2 above)
Comment (DrogoNevets) [reply src/server/cph-number/controller.js:34]: totally agree!!

File: src/server/exit-page/index.njk (hunk 3 above)
Comment (DrogoNevets) [reply src/server/exit-page/index.njk:8]: hmmm woder where i got this from....will have a look

File: src/server/common/constants/status-codes.js (hunk 1 above)
Comment (hughfdjackson) [reply src/server/common/constants/status-codes.js:12]: @DrogoNevets  - yup, 100% agreed that 302 is the right choice here.  It's just about the wording of the status code in the map.

Comment (DrogoNevets) [reply src/server/common/constants/status-codes.js:12]: oh i see, can change it to `Found` as per MDN?

File: src/server/exit-page/index.njk (hunk 3 above)
Comment (DrogoNevets) [reply src/server/exit-page/index.njk]: now follow [this layout](https://design-system.service.gov.uk/styles/layout/)

**Review by hughfdjackson — APPROVED** [review]

**Comment by eoin-corr-git** [comment]
Page title should be 'This service is not available for your movement type'

**Pull request merged**